.contributing/the-process.md

```diff
@@ -125,4 +154,24 @@ Notes on the various tools 
 
 -   Google docs are great for creating longer, more complex documents (eg. with diagrams and a TOC)
 
--   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
\ No newline at end of file
+-   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* - User Flow files - *Section* (CL)"
```

Comment:
If we go with "<" over "-" we'll need to change here too

Reply:
done!